src/main/java/me/rayzr522/jsonmessage/JSONMessage.java

```diff
@@ -671,9 +722,25 @@ static Object createTextPacket(String message) {
                 e.printStackTrace();
                 return null;
             }
-
+            
         }
-
+        
+        static void sendTextPacket(String message, Player... players) {
+            try {
+                for (Player player : players) {
+                    Class chatTypeClass = getClass("{nms}.ChatMessageType");
```

Comment:
all pre-computable reflection should be moved to the static initializer of the inner Reflection class, doing this each time `sendTextPacket` is called not to mention inside of a potentially high-iteration for-loop could lead to some pretty serious performance issues D:

Reply:
The reflection stuff isn't from me, so you have to ask @darbyjack here.
I have 0 knowledge with reflection stuff.